### ISSUE TITLE
Widevinecdm component error Mac

### ISSUE THREAD

**Issue by alterlove** [nirewen/discord-netflix#63]
I get this error all time I try to play any title, I tried some fix on internet, but it doesn't work. I tried to delete the folder of the component and forcing chrome to download it again, but nothing, still the same. Any way to fix it?
![captura de pantalla 2019-02-26 a las 4 04 37](https://user-images.githubusercontent.com/38593578/53384679-12769500-397c-11e9-84e4-29892be3843b.png)

**Comment by nirewen** [comment]
That's weird... I thought having 7.0.0 for Mac would solve this. Just in case, are you using the latest version?

**Comment by alterlove** [comment]
I'm using High Sierra with latest updates

**Comment by nirewen** [comment]
Is this still happening on version 2.2.0? If so, can you try version 2.0.1?

**Comment by stormdragon288** [comment]
Also getting the same error. On Mac OS X 10.14

**Comment by nirewen** [comment]
@stormdragon288 2.0.3 works for you?

**Comment by stormdragon288** [comment]
> @stormdragon288 2.0.3 works for you?

Gives me the same error in 2.0.3

**Comment by alterlove** [comment]
@stormdragon288 does it works in Safari too? I get Silver right error in safari but with the same error code.

**Comment by stormdragon288** [comment]
> @stormdragon288 does it works for you in Safari too? I get Silver right error in safari but with the same error code.

Yeah safari's working fine. This stand alone app however does not work (for whatever reason)

**Comment by alterlove** [comment]
Did you updated your safari to 12? Cuz apple says it works in old versions.

**Comment by stormdragon288** [comment]
> Did you updated your safari to 12? Cuz apple says it works in old versions.

Yes! on 12 and Netflix works.

**Comment by nirewen** [comment]
Maybe WidevineCDM updated and the module doesn't have the new binaries as it's not mantained anymore... Maybe using a different Electron version it works again, but I can't know as I don't have a Mac to keep testing.

**Comment by lollilol** [comment]
I have this error too on Linux. Im using Manjaro and have installed chromium-widevine from AUR and its working in Chromium.
I git pulled the latest source and compiled it. Tryed with the app image and the linux-unpacked.

**Comment by itsjakeb** [comment]
I also have the same error.
![image](https://user-images.githubusercontent.com/20141653/54870935-d2e97000-4da4-11e9-91be-ff8932b7396d.png)

**Comment by MeetPing** [comment]
Linux also has that error missing widevine and in chrome://components its not found

**Comment by MJ94** [comment]
I can confirm this issue on macOS 10.14.5.

**Comment by renanrcp** [comment]
I think this can be a solution? https://github.com/quanglam2807/electron-widevinecdm/issues/8 I'll fork and try later

**Comment by MoarcyVega** [comment]
I got the same error on Ubuntu 20.04. 
In Chrome and Firefox Netflix works fine.
